.taskcluster.yml

```diff
@@ -140,7 +143,8 @@ tasks:
           /home/build-user/DeepSpeech/tf/tc-apt.sh &&
           npm update && npm install -g node-gyp node-pre-gyp &&
           sudo -H -u build-user /bin/bash /home/build-user/DeepSpeech/ds/tc-build.sh &&
-          sudo -H -u build-user /bin/bash /home/build-user/DeepSpeech/ds/tc-package.sh
+          sudo -H -u build-user /bin/bash /home/build-user/DeepSpeech/ds/tc-package.sh &&
+          sudo -H -u build-user TASK_ID=${TASK_ID} GITHUB_HEAD_REPO_URL=${GITHUB_HEAD_REPO_URL} GITHUB_HEAD_SHA=${GITHUB_HEAD_SHA} EVENT_HEAD_USER_EMAIL={{ event.head.user.email }} EVENT_HEAD_REPO_URL={{ event.head.repo.url }} /bin/bash /home/build-user/DeepSpeech/ds/tc-schedule-tasks.sh
```

Comment:
Only needed in PR, this will schedule training of LDC93S1 and C++/Python/Node tests, no useful "production" artifact.